index.html

```diff
@@ -1381,6 +1381,47 @@
       </section>
     </section>
 
+    <section>
+      <h2>Verification</h2>
+
+      <p>
+This section describes a number of checks required to verify a claim.
+Some checks are essential for all verifiable claims, while some are applicable to only some claims.
+      </p>
+      <section>
+        <h3>Structural Validity</h3>
+        <p>
+          <ul>
+            <li>Document is valid JSON-LD.</li>
+            <li>Required properties are present. For example, for a Credential, <code>id</code>, <code>type</code>, and <code>claim</code> are required.</li>
+            <li>Property values match expectations described in the specification. For example, the document type for a verifiable claim must contain the class "Credential".</li>
```

Comment:
Wrap all lines to 75 characters (try to follow the format of the rest of the document). It is inevitable that we'll need to reformat the spec at some point because of inconsistencies that sneak in, but we try to more or less maintain the same format throughout the spec.